packages/upset/src/components/Columns/Attribute/AttributePlots/StripPlot.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { useRecoilValue } from 'recoil';
+import { ScaleLinear } from 'd3-scale';
+import { Aggregate, SixNumberSummary, Subset } from '@visdesignlab/upset2-core';
+import { dimensionsSelector } from '../../../../atoms/dimensionsAtom';
+import { visibleAttributesSelector } from '../../../../atoms/config/visibleAttributes';
+
+type Props = {
+  scale: ScaleLinear<number, number, never>;
+  values: number[];
+  attribute: string;
+  summary: SixNumberSummary;
+  isAggregate: boolean;
+  row: Subset | Aggregate;
+};
+
+// Dot plot component for the attributes plots
```

Comment:
Make this a JSDoc comment, maybe more detail, and should it say `// Strip plot component...`?

Reply:
Yes, it should. I copied the dotplot file over to start this, as they largely have the same infrastructure.